app/src/routes/constants.js

```diff
@@ -115,13 +120,8 @@ export const pageRendering = {
   },
   PROJECT_USERDEBUG_PAGE: { component: LaunchesPage, layout: ProjectLayout },
   PROJECT_USERDEBUG_TEST_ITEM_PAGE: { component: TestItemPage, layout: ProjectLayout },
-  PROJECTS_PAGE: {
-    component: ProjectsPage,
-    layout: InstanceLayout,
-    rawContent: true,
-  },
   [PROJECT_DETAILS_PAGE]: {
-    component: ProjectsPage,
+    component: () => '!!!!!!!!!!!!!!',
```

Comment:
Why not to remove the `PROJECT_DETAILS_PAGE` as well?
It was used as a part of previous projects page only.

Reply:
Done.